docs/extensions/tutorials/_posts/1970-01-02-WritingATheme.md

```diff
@@ -281,17 +329,18 @@ Open `Style` tab and choose `Customize theme`. Theme variables are grouped into
     // other variables
   },
   "layout": {
-    "sections": [{
+    "sections": [
+    {
       "title": "Restaurants",
       "properties": ["subtitleColor"]
-    }, {
+    },
       // other sections
-    }]
+    ]
   }
 }
 ```
 
-The only thing left to do is to use this variable in theme file:
+The only thing left to do is to use this variable in the theme file. Again, search for the "export default" statement:
```

Comment:
Use "\`" instead of normal quotes. So, it's formatted like this: `export default`.